### ISSUE TITLE
Selecting numbers in tgui is clunky

### ISSUE THREAD

**Issue by Justice12354** [tgstation/tgstation#81379]
Reporting client version: 515.1630

## Issue Summary
When selecting numbers in the tgui (namely "Edit Filters" and "Edit Color as Matrix" panels), you have to press M1 again to be able to edit the number in the slot.
You can only "drag" the values in that slot up or down before selecting it.

## Round ID:
[223646](https://scrubby.melonmesa.com/round/223646)

## Reproduction:
-Open Edit Color Matrix Panel;
-Click on a random number to edit it;
-The number turns white and typing doesn't change it. Furthermore, you can no longer drag the slot up or down with your mouse to change the value without typing it;
-Click again on said slot to select it and then type in it.

Basically, you gotta click twice in order to edit numbers in some tgui

The thick red lines are around the "selected" slots (which have to be clicked again to edit), and the thin green lines are around the non-selected slots:
![imagem](https://github.com/tgstation/tgstation/assets/42555530/405579f9-2d64-4a1c-bc4e-363a14d1ea27)
![imagem](https://github.com/tgstation/tgstation/assets/42555530/2c9eb7ed-2df0-4903-aab6-945142e24bed)

**Comment by SyncIt21** [comment]
Duplicate #80971
